android/java/org/chromium/chrome/browser/settings/AppearancePreferences.java

```diff
@@ -250,21 +251,16 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
         String key = preference.getKey();
         boolean shouldRelaunch = false;
         if (BravePreferenceKeys.BRAVE_BOTTOM_TOOLBAR_ENABLED_KEY.equals(key)) {
-            SharedPreferences prefs = ContextUtils.getAppSharedPreferences();
             Boolean originalStatus = BottomToolbarConfiguration.isBraveBottomControlsEnabled();
             updatePreferenceSummary(
                     BravePreferenceKeys.BRAVE_BOTTOM_TOOLBAR_ENABLED_KEY,
                     !originalStatus ? R.string.text_on : R.string.text_off);
-            prefs.edit()
-                    .putBoolean(
-                            BravePreferenceKeys.BRAVE_BOTTOM_TOOLBAR_ENABLED_KEY, !originalStatus)
-                    .apply();
+            ChromeSharedPreferences.getInstance().writeBoolean(
+                    BravePreferenceKeys.BRAVE_BOTTOM_TOOLBAR_ENABLED_KEY, !originalStatus);
```

Comment:
No functional difference between the old code and this new code, right @SergeyZhukovsky?

I'm guessing this set off a pre-submit / lint error based on follow up:
https://github.com/brave/brave-core/pull/30600/commits/1c56cd26f05c4d02ece0373506b19866e4161de9